src/Drawing.ts

```diff
@@ -645,7 +645,7 @@ export const render: (drawing: Drawing) => (ctx: CanvasRenderingContext2D) => IO
         return (ctx) =>
           pipe(
             readOnlyArrayTraverseIO(drawing.drawings, (d) => go(d)(ctx)),
-            IO.chain(() => IO.of(ctx))
+            IO.chain(() => () => ctx)
```

Comment:
this is

```ts
IO.map(() => ctx)
```

Reply:
See https://github.com/gcanti/graphics-ts/pull/7/commits/d3848bcfd2b47782484a1da84fe31d5e2cae11f8